doc/nrf/gs_ins_windows.rst

```diff
@@ -183,6 +191,11 @@ To switch to the latest state of development, enter the following commands::
    git checkout master
    west update
 
+.. note::
+   Every time you pull, rebase, check out a branch or in general change or
```

Comment:
```suggestion
   Run ``west update`` every time you change or modify the current working branch (for example, when you pull, rebase, or check out a different branch).
   This will bring the project repositories to the matching revision defined by the manifest file.
```